Templates/TypeScript/EnumType.stencil

```diff
@@ -1,6 +1,4 @@
-{% if header %}{{ header }}
-
-{% endif %}export enum {{ enumType.name }} {
+{% include "Helpers/Header.stencil" %}export enum {{ enumType.name }} {
```

Comment:
Is it possible to have the export statement on the next line? Same for the other files